### PR TITLE
Add version information and cmake install configuration

### PR DESCRIPTION
**Pull request by MrDmitry**

### CODE REVIEW

**Review by ilia-glushchenko — APPROVED** [review]

File: Arion/ArionVersionInfo.rc.in [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include <Windows.h>
```
Comment (ilia-glushchenko) [inline Arion/ArionVersionInfo.rc.in:9]: Copyrights?

**Review by ilia-glushchenko — CHANGES_REQUESTED** [review]

File: Arion/ArionVersionInfo.rc.in (hunk 1 above)
Comment (MrDmitry) [reply Arion/ArionVersionInfo.rc.in:9]: Done

**Comment by ilia-glushchenko** [comment]
Please open the issue and add its index to the commit message.

**Comment by MrDmitry** [comment]
Done

**Review by ilia-glushchenko — APPROVED** [review]

**Review by krupitskas — APPROVED** [review]

**Pull request merged**